ui/src-tauri/src/commands/app.rs

```diff
@@ -15,6 +20,23 @@ pub fn close_main_window(app_handle: tauri::AppHandle) {
     }
 }
 
+#[tauri::command]
+pub fn open_tray(app_handle: tauri::AppHandle) {
+    let window = get_main_window(&app_handle);
+    if let Ok(true) = window.is_visible() {
+        let _ = window.hide();
+    } else {
+        let _ = window.show();
+        let _ = window.move_window(Position::TopRight);
```

Comment:
```suggestion
        let _ = window.move_window(Position::TrayCenter);
```
This should work

Reply:
Actually this breaks on windows, I guess the tray icon is not created and it tries to get its position to align it self.

Reply:
Yeah, I remember bumping into the problem that we can't get tray coordinates before we have initialized the tray icon...